### PR TITLE
✨ Improve styleguide

### PR DESCRIPTION
**Pull request by artisologic**
- Extract as includes
- Componentize CSS
- Reuse as much as possible existing atomic classes

### CODE REVIEW

File: _includes/styleguide/grid.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		<code class="col xs-1"></code>
+	</div>
+
+	<div class="col">
+		<h3 class="Styleguide-heading">Responsive</h3>
+	</div>
+
+	<div class="grid mb-2">
+		<code class="col xs-12 md-6 lg-5"></code>
+		<code class="col xs-12 md-6 lg-7"></code>
```
Comment (peduarte) [inline _includes/styleguide/grid.html]: how come you're using `code` tag here? does it look like a code block or does it still look like a grid?

File: _includes/styleguide/typography.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	<code>&lt;h3&gt;</code>
+	<h3 class="mb-3">The quick brown fox jumps over the lazy dog.</h3>
+
+	<code>&lt;h4&gt;</code>
+	<h4 class="mb-3">The quick brown fox jumps over the lazy dog.</h4>
+
+	<code>&lt;h5&gt;</code>
+	<h5 class="mb-3">The quick brown fox jumps over the lazy dog.</h5>
+
+	<code>&lt;h6&gt;</code>
```
Comment (peduarte) [inline _includes/styleguide/typography.html]: I think `code` here makes sense 👍

File: _includes/styleguide/grid.html (hunk 1 above)
Comment (artisologic) [inline _includes/styleguide/grid.html]: Looks like both :)

Comment (peduarte) [inline _includes/styleguide/grid.html]: weird, never seen a grid with `block` but fair enough!

Comment (artisologic) [inline _includes/styleguide/grid.html]: I can put the `<code>` inside divs if you want but then makes it harder to auto-generate the content.

**Pull request merged**